README.md

```diff
@@ -100,6 +100,7 @@ Some env options are available for use this interface for **only one server** (w
 - `THEME_*`: See table in [Theme options](#theme-options) section (see [#283](https://github.com/Joxit/docker-registry-ui/pull/283)). Since 2.4.0
 - `TAGLIST_ORDER`: Set the default order for the taglist page, could be `num-asc;alpha-asc`, `num-desc;alpha-asc`, `num-asc;alpha-desc`, `num-desc;alpha-desc`, `alpha-asc;num-asc`, `alpha-asc;num-desc`, `alpha-desc;num-asc` or `alpha-desc;num-desc` (see [#307](https://github.com/Joxit/docker-registry-ui/pull/307)). (default: `alpha-asc;num-desc`). Since 2.5.0
 - `CATALOG_DEFAULT_EXPANDED`: Expand by default all repositories in catalog (see [#302](https://github.com/Joxit/docker-registry-ui/issues/302)). (default: `false`). Since 2.5.0
+- `TAGS_PER_PAGE`: Number of specific image tags shown per page. (default: `100`). 
```

Comment:
I'm not sure how you add a default value

Reply:
The default value is already working thanks to this (I like when I'm one step ahead):
https://github.com/Joxit/docker-registry-ui/blob/d2e6cdcab16c253ae14135d3af38c331953a44f7/src/scripts/utils.js#L93-L111